bpf/lib/maps.h

```diff
@@ -99,6 +99,21 @@ struct {
 } AUTH_MAP __section_maps_btf;
 #endif
 
+#ifdef CONFIG_MAP
+/*
+ * CONFIG_MAP is an array containing runtime configuration information to the
+ * bpf datapath.  Each element in the array is 64 bytes, initially use of only
+ * one array element at index 0 is defined.
+ */
+struct {
+	__uint(type, BPF_MAP_TYPE_ARRAY);
+	__type(key, __u32);
+	__type(value, struct runtime_config_0);
+	__uint(pinning, LIBBPF_PIN_BY_NAME);
+	__uint(max_entries, CONFIG_MAP_SIZE);
+} CONFIG_MAP __section_maps_btf;
```

Comment:
Yes. Is there a flag for that? The only reference I could find from the datapath code is this:
```
tests/pktgen.h: * with kernels before 5.2 that lack read-only map support. This test suite
```

Reply:
`BPF_F_RDONLY_PROG` is your friend!

Reply:
This didn't work out for some reason?

Reply:
We have not defined that flag in userspace, nor are we using it anywhere in the datapath. I don't know what happens on older kernels not supporting this flag if it is set, and don't feel like this PR is the right place to experiment on that. I feel like this would be a separate piece of work to intro this flag (and maybe others) on the userspace side, make use of it on all the maps that are read only from the datapath side, and deal with the kernel versions, if need be.